torch/_inductor/codegen/wrapper.py

```diff
@@ -334,6 +335,59 @@ def call(args):
                 )
             V.graph.sizevars.codegen(self.wrapper_call, V.graph.graph_inputs)
 
+    def write_mkldnn_param_gen_utils(self):
+        self.header.splice(
+            """
+            class mkldnn_param_cache:
+                def __init__(self):
+                    self.p_cache = dict()
+
+                def add_param(self, param_id, param, param_type):
+                    self.p_cache[param_id] = (param, param_type)
+
+                def get_param(self, param_id):
+                    if param_id in self.p_cache:
+                        return self.p_cache[param_id][0]
+                    else:
+                        return 0
+
+                def check_param(self, param_id):
+                    if param_id in self.p_cache:
+                        return True
+                    else:
+                        return False
```

Comment:
Seems we can use `get_param(param_id) != 0` to check the existence in cache too?

Reply:
I'd make the logic to be direct that check_param for judgement and get_param to obtain value.